doc/collection/service/api.md

```diff
@@ -264,3 +263,13 @@ class OrganizationService extends CollectionService<Organization> {
   }
 }
 ```
+
+## Formatters
+You can format your data when it comes from Firestore with a custom function.
+To do so you have to override the function `formatFromFirestore`.
+```typescript
+  formatFromFirestore(stakeholder: Stakeholder) {
```

Comment:
Maybe specify that the value is readonly : 
`formatFromFirestore(stakeholder: Readonly<Stakeholder>)`